src/runtime_src/xdp/profile/plugin/aie_debug/aie_debug_plugin.cpp

```diff
@@ -331,15 +339,15 @@ namespace xdp {
     xclGetDebugIPlayoutPath(handle, pathBuf, PATH_LENGTH);
 
     std::string sysfspath(pathBuf);
-    uint64_t deviceId = db->addDevice(sysfspath); // Get the unique device Id
+    uint64_t device_id = db->addDevice(sysfspath); // Get the unique device Id
```

Comment:
Please change this back to camel case to be consistent with rest of file.

Reply:
this change was *for* consistency